documentation/homework/week1/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import './App.css';
+import Header from './components/header';
+import TodoList from './components/todoList';
```

Comment:
Your component files should be named exactly the same as the components they represent. In this case, that means capitalising your js files to be `TodoList.js` and `TodoAdd.js`, etc.